src/main/resources/de/qabel/desktop/ui/layout.fxml

```diff
@@ -26,29 +26,17 @@
                              </Pane>
                             <VBox BorderPane.alignment="CENTER" styleClass="aliasContainer">
                                <children>
-                                  <Label fx:id="alias" text="Guy Pearce Longerman">
-                                     <font>
-                                        <Font name="System Bold" size="13.0" />
```

Comment:
I understand why you remove the font name, but it still needs to be bold and bigger than the rest (according to the mockups). I think this makes it worse than wrong font but bold and big.